### ISSUE TITLE
Help text - misleading text under Dates and gridreferences?

### ISSUE THREAD

**Issue by DavidRoy** [BiologicalRecordsCentre/brc-ds-eval#6]
Within the 'Dates and grid references' section there is text:
The tools do not support the use of vague dates or date ranges - they work with dates specified in either of these two formats:

This seems misleading as vague dates or date ranges could be loaded and viewed (i.e. via record details) but they do not get used in the visualisations.  Suggest rewording the help to:
Records with vague dates or date ranges can be loaded within the tool but data with imprecise dates are not supported within visualisations (e.g. phenology plots).

**Comment by burkmarr** [comment]
Actioned.
